starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,49 @@
+.anything {
+  background-image: url(/Users/mgad/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg);
```

Comment:
Don't use this kind of path as they will only work on your computer (because you are starting from the Roots)
Instead use the folder path like this : 

"../images/apple-pie.jpg"

It will works on every computer as it's only related to the project folder (same for your stylesheet) :)